### PR TITLE
Fixed Dockerfiles and Makefile. fixes issue #28

### PR DESCRIPTION
**Pull request by devang-gaur**
fabric8-hdd-bug-id: fabric8-hdd/supervisor#28
Signed-off-by: dev-gaur dgaur@redhat.com

### CODE REVIEW

File: run_tests.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+TEMP=$(date +%s)
+
+docker build -t supervisor-tests:$TEMP -f Dockerfile.tests --build-arg CACHEBUST=$TEMP .
+
```
Comment (anmolbabu) [inline run_tests.sh:8]: Building the image here and deleting it below?

Comment (anmolbabu) [reply run_tests.sh:8]: Can we have something useful in between however minimalistic possible?

**Review by anmolbabu — CHANGES_REQUESTED** [review]

File: run_tests.sh (hunk 1 above)
Comment (devang-gaur) [reply run_tests.sh:8]: @anmolbabu Useful, as in? I think a lot of steps will get stuffed in gradually as we move ahead.

Comment (anmolbabu) [reply run_tests.sh:8]: probably run the supervisor-tests container and collect success or failure(if possible?) at this moment?

Comment (anmolbabu) [reply run_tests.sh:8]: I understand that build time itself, tests would be run but if you see https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/runtest.sh#L72 I feel its for some specific reason related to observing post deployment conditions. Now, if its just to get db setup and run tests against it I am ok with not having it but please check the purpose of it with @miteshvp

**Pull request closed without merging**